statisticsapp/src/main/java/com/example/statisticsapp/presentation/ui/statistics/adapter/TasksStatisticsViewHolder.kt

```diff
@@ -0,0 +1,41 @@
+package com.example.statisticsapp.presentation.ui.statistics.adapter
+
+import com.example.statisticsapp.R
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import com.example.plantsapp.domain.model.Task
+import com.example.statisticsapp.databinding.ItemTaskAndCompletionsNumberBinding
+import com.example.statisticsapp.presentation.ui.statistics.StatisticsViewModel
+
+class TasksStatisticsViewHolder(
+    private val binding: ItemTaskAndCompletionsNumberBinding
+) : RecyclerView.ViewHolder(binding.root) {
+
+    fun bind(taskStatisticsInfo: StatisticsViewModel.TaskStatisticsInfo) {
+        with (binding) {
+            tvTaskAndCompletionNumber.text = root.context.getString(
+                taskStatisticsInfo.task.getTitleRes(),
+                taskStatisticsInfo.amount
+            )
+        }
+    }
+
+    private fun Task.getTitleRes(): Int {
+        return when (this) {
+            is Task.WateringTask -> R.string.title_watering_task
+            is Task.SprayingTask -> R.string.title_spraying_task
+            is Task.LooseningTask -> R.string.title_loosening_task
+            is Task.TakingPhotoTask -> R.string.title_taking_photo_task
+        }
+    }
```

Comment:
Just for your info (right now it is not needed), but you can create separate ui-core module with resources, styles and so on, and reuse icons styles and even layouts 

But again, no need to do it right now

Reply:
> Just for your info (right now it is not needed), but you can create separate ui-core module with resources, styles and so on, and reuse icons styles and even layouts
> 
> But again, no need to do it right now

that's cool!